GoogleCloudPlatform/pgadapter
# chore(cleanup): static imports + expected first

Clean up the ProtocolTest class to make all tests consistent:
1. Use static imports for commonly used test methods.
2. Make sure the expected argument is first.
3. Remove warnings for deprecated methods.
4. Remove warnings for unused variables and method return values.
